openapi-v1.yaml

```diff
@@ -3584,6 +3584,57 @@ paths:
               message: "Error try again"
               code: 1
 
+  /arrays/{namespace}/{array}/evolve:
+    parameters:
+      - name: namespace
+        in: path
+        description: namespace array is in (an organization name or user's username)
+        type: string
+        required: true
+      - name: array
+        in: path
+        description: name/uri of array that is url-encoded
+        type: string
+        required: true
+    post:
+      description: evolve array schema by adding an attribute
+      tags:
+        - array
+      operationId: addAttribute
+      parameters:
+        - name: attribute
+          in: body
+          description: attribute to evolve
+          schema:
+            $ref: "#/definitions/Attribute"
+          required: true
+      responses:
+        204:
+          description: schema evolved successfully
+        default:
+          description: error response
+          schema:
+            $ref: "#/definitions/Error"
+    delete:
```

Comment:
There should just be a post, this is like a query in that the posted schema evolution object should contain all the details and the server just calls `evolve`